core/vm/actor/builtin/v4/todo.hpp

```diff
@@ -1,25 +0,0 @@
-/**
```

Comment:
Why? It was working.  
I would keep assertion, rather than debugging unexpected/silent errors.